cockroachcloud/monitoring-page.md

```diff
@@ -1,19 +1,120 @@
 toc: true
 docs_area: manage
 ---
 
-The **Monitoring** page is currently under development. In the future, time series charts will display here. In the meantime, you can use the cluster's built-in DB Console to see them.
+The **Monitoring** page is accessible on {{ site.data.products.dedicated }} clusters. This page allows you to:
 
-You can use the cluster's built-in DB Console to view time series data on SQL queries, troubleshoot query performance, view a list of jobs, and more.
+- Set up cluster [monitoring with Datadog](#monitor-with-datadog).
+- Access the cluster's [built-in DB Console](#access-the-db-console) to view time-series data on SQL queries, troubleshoot query performance, view a list of jobs, and more.
+
+To view the **Monitoring** page, [log in](https://cockroachlabs.cloud/) and click **Monitoring**.
+
+## Monitor with Datadog
+
+The {{ site.data.products.dedicated }} integration for Datadog enables data collection and alerting on [CockroachDB metrics](https://docs.datadoghq.com/integrations/tk) using the Datadog platform.
+
+To set up Datadog monitoring with {{ site.data.products.dedicated }}, your Datadog account must be associated with a [Datadog organization](https://docs.datadoghq.com/account_management/#organizations).
+
+{{site.data.alerts.callout_success}}
+For more information about using Datadog, see the [Datadog documentation](https://docs.datadoghq.com/).
+{{site.data.alerts.end}}
+
+### Enable integration
+
+To enable Datadog monitoring for a {{ site.data.products.dedicated }} cluster:
+
+1. On the cluster's **Monitoring** page, click **Setup** in the **Datadog** panel.
+
+1. Fill in the **API key** and **Datadog Site** fields with the corresponding values.
+    - The **API key** is associated with your Datadog organization. If you don't have an API key to use with your {{ site.data.products.dedicated }} cluster, you need to create one. For instructions, see the [Datadog documentation](https://docs.datadoghq.com/account_management/api-app-keys/#add-an-api-key-or-client-token).
+    - Your **Datadog Site** corresponds to your Datadog URL. For more details, see the [Datadog documentation](https://docs.datadoghq.com/getting_started/site/).
+
+1. Click **Create**. The integration will take approximately 15 seconds to 2 minutes to enable, depending on the size of your cluster and the current load on the system.
+
+1. When enabled, the **Integration status** in the panel will show as `Active`.
+
+    The cluster will also appear on your Datadog [Infrastructure List](https://docs.datadoghq.com/infrastructure/list/) once it is registered on Datadog. This can take up to several minutes.
+
+### Verify integration status
+
+The **Integration status** in the **Datadog** panel reports the following:
+
+- `Active` indicates that the integration has been successfully deployed.
+- `Inactive` indicates that the integration has not been successfully deployed.
```

Comment:
> `Inactive` indicates that the integration has not been successfully deployed.

Does the wording here indicate a problem? If the customer never tried setting up the integration in the first place, then "has not been **successfully** deployed" feels like it indicates some error/failure occurred, when the reality is that they just never tried deploying it before. 

However, it's indeed also true that if an error occurred during the setup, the `Inactive` status will show as well, where the error is indicated by the "Datadog integration rollout failed. Retry rollout" text. How can we word this to cover both scenarios? 🤔

Reply:
Good point - how about:

"Setup has either not been attempted or has encountered an error."